pkg/config/cache.go

```
@@ -0,0 +1,108 @@
+	}
+
+	return filepath.Join(cacheDir, "cache.yaml"), nil
+}
+
+func withCacheFileLock(cacheFile string, fn func() error) error {
+	lock := flock.New(cacheFile)
+	err := lock.Lock()
+	if err != nil {
+		return errors.Wrap(err, "error acquiring file lock")
+	}
+	defer lock.Unlock()
```

golangci-lint (1.62.2)

Error return value of `lock.Unlock` is not checked

(errcheck)